src/components/HappinessNumber.tsx

```diff
@@ -75,37 +83,53 @@ export default function HappinessNumber({
       <img src={(change > 0 ? ArrowUpIcon : ArrowDownIcon) as any} />
     </div>
   );
-  return (
-    <Column className=" w-full items-center">
-      {editable && <Changer change={0.5} />}
-      <div className=" h-3" />
-      <input
-        type="text"
-        value={currentHappiness === -1 ? "--" : currentHappiness.toFixed(1)}
-        className="h-auto max-w-[80px] resize-none border-0 border-gray-400 bg-transparent p-0 text-center text-4xl font-medium focus:border-b-1 focus:outline-none"
-        onChange={(e) => {
-          if (e) {
-            clearTimeout(updateHappinessTimeout.current);
-            let happinessNum = 0;
-            if (e.target.value !== "") {
-              happinessNum = parseFloat(e.target.value);
-              if (happinessNum < 0) {
-                happinessNum *= -1;
-              }
-              // While loop is unncessary but does make absolutley sure it is in range
-              while (happinessNum > 10) {
-                happinessNum /= 10;
-              }
+
+  const numInput = (
```

Comment:
You should use the same style for both inner components in this file. Remove the inconsistency with camel vs pascal case. We've been using pascal case for components so please continue to follow this style unless you have a good reason to use camel case here in which case tell me. Also this should be in the functional style as a component, not just a variable.